src/ClusterBootstrap/deploy.sh

```diff
@@ -6,14 +6,16 @@ rm -rf deploy/!(bin) cloudinit* !(config).yaml
 ./cloud_init_deploy.py clusterID
 # render a machine list which is to be deployed
 ./cloud_init_aztools.py prerender
+# deploy azure resources except for VMs
+./cloud_init_aztools.py -v deployframework
 # render templates and prepare binaries
 ./cloud_init_deploy.py render
 # pack files for cloudinit into a tar file which would be pushed to cloudinit docker
 ./cloud_init_deploy.py pack
 # push dockers
 ./cloud_init_deploy.py docker servicesprerequisite
-# execute a deployment action based on the rendered files and machine list got in prerender
-./cloud_init_aztools.py -v deploy
+# execute a deployment action to deploy VMs based on the rendered files and machine list got in prerender
+./cloud_init_aztools.py -v addmachines
```

Comment:
Is this an intended change?

Reply:
yes, since we now first deploy framework (resource group, nsg, storage account etc.) to get storage account connection string, then render, and finally deploy VMs.